CMakeLists.txt

```diff
@@ -1435,6 +1435,14 @@ if(BUILD_TESTS)
                       --max-writes-ahead 1000 --repetitions 10000
     )
 
+    add_perf_test(
+      NAME pi_basic
```

Comment:
What does "pi" stand for here?

Reply:
piccolo, it's what we use elsewhere too.